src/area-chart/__tests__/area-chart-use-chart-model.test.tsx

```diff
@@ -8,7 +8,7 @@ import { ChartDataTypes } from '../../internal/components/cartesian-chart/interf
 import { act, render, fireEvent } from '@testing-library/react';
 import { AreaChartProps } from '../interfaces';
 import { KeyCode } from '../../internal/keycode';
-import { useReaction } from '../model/async-store';
+import { useReaction } from '../../internal/async-store';
```

Comment:
We discussed it in the past: https://github.com/cloudscape-design/components/pull/998

You have not done your part of formalizing this utility, so it also cannot be moved

Reply:
The tool is already used in both area-chart and table. Why do you think keeping it under area-chart is better?

Reply:
The sticky-columns util does not use the async-store utility to its full extent. Would it work for you if I update the util to use a simplified store and move that one to under internal?

Reply:
I'll move the async-store back to under src/area-chart to unblock the PR.

Created a new PR to address the feedback on async store: https://github.com/cloudscape-design/components/pull/1246